src/libraries/System.Runtime/tests/System.Reflection.Tests/TypeTests.cs

```diff
@@ -0,0 +1,258 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Linq;
+using Xunit;
+
+namespace System.Reflection.Tests
+{
+    public static unsafe partial class TypeTests
+    {
+        [Theory]
+        [InlineData(typeof(int), "System")]
```

Comment:
We have a bunch of Namespace tests at https://github.com/dotnet/runtime/blob/99cbed26c35da4ddb084011ffbb32dd6691ceee4/src/libraries/System.Runtime/tests/System.Reflection.Tests/TypeInfoTests.cs#L1479-L1490  . (TypeInfo and Type are the same thing - they got split in early days of .NET Core, but it got reverted.) 

Should we just add the missing cases there?

Reply:
> (TypeInfo and Type are the same thing - they got split in early days of .NET Core, but it got reverted.)

Didn't realize they were reconciled. I originally had it in here, but saw it was different so created a suite of new tests. I'm happy to move them here.